src/test/scala/org/allenai/deep_qa/data/OmnibusDaDatasetReaderSpec.scala

```diff
@@ -0,0 +1,46 @@
+package org.allenai.deep_qa.data
+
+import com.mattg.util.FileUtil
+import org.scalatest._
+
+class OmnibusDaDatasetReaderSpec extends FlatSpecLike with Matchers {
+
+  val fileUtil = new FileUtil
+  val questionText1 = "Coal is a nonrenewable energy resource. Identify the original "+
+    "source of the energy stored in coal."
+  val answerText1 = "The Sun is the original source of energy."
+  val header = "\"id\",\"parentId\",\"isArchived\",\"questionText\"," +
```

Comment:
I would use triple-quoted string literals here; it would make this a lot easier to read.